ansible/roles/xroad-ss/tasks/rhel.yml

```diff
@@ -1,9 +1,6 @@
 ---
 - name: install xroad security server packages
-  yum:
-    name: xroad-securityserver-fi
-    state: present
-    update_cache: yes
+  command: yum -y install xroad-securityserver-fi
```

Comment:
The original yum-module task includes `update_cache: yes`. If replacing that with manual install, is there / should there be a manual cache update as well?

Reply:
as I understand, yum makecache fast (in xroad-base) updates the cache